Clicker/ViewControllers/DraftsViewController+Extension.swift

```diff
@@ -18,18 +18,30 @@ extension DraftsViewController: DraftCellDelegate {
 extension DraftsViewController: ListAdapterDataSource {
     
     func objects(for listAdapter: ListAdapter) -> [ListDiffable] {
-        return drafts
+        if drafts.count > 0 {
+            return drafts
+        }
+        return [EmptyStateModel(type: .draftsViewController(delegate: self))]
```

Comment:
shorten to `return drafts.count > 0 ? drafts : [EmptyStateModel(type: .draftsViewController(delegate: self))]`